advanced-examples/system-performance/stressng-pcp/README.md

```diff
@@ -0,0 +1,181 @@
+# Stress-NG + PCP Workflow
+
+## Workflow Description
+
+The primary [`workflow.yaml`](workflow.yaml) is designed to collect system metadata with
+Ansible [gather facts](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/gather_facts_module.html)
+using the Arcaflow
+[metadata plugin](https://github.com/arcalot/arcaflow-plugin-metadata), and then to run
+loops of a stress-ng sub-workflow with various parameters.
+
+In addition to the stress-ng workload, the sub-workflow collects system metrics with
+[Performance Co-pilot](https://pcp.io/) using the Arcaflow
+[pcp plugin](https://github.com/arcalot/arcaflow-plugin-pcp).
+
+This is a complex workflow that exercises many Arcaflow features, including
+[sub-workflows](/basic-examples/sub-workflow-foreach),
+[serial wait-for](/basic-examples/serial-wait_for),
+[namespaced scopes](/basic-examples/namespaced-scopes),
+and [bind constants](/basic-examples/bind-constants).
+
+## Files
+
+- [`workflow.yaml`](workflow.yaml) -- Defines the outer workflow input schema, the
+plugins and sub-workflow to run and their data relationships, and the output to present
+to the user
+- [`stressng-workflow.yaml`](stressng-workflow.yaml) -- Defines the inner workflow input
+schema, plugins, and output for stress-ng tests. This workflow is looped over by the
+`workflow.yaml`, but it can also be used stand-alone
```

Comment:
I think the word "workflow" should follow the references to the workflow files. "the `workflow.yaml`" sounds wrong because it's specifying one thing. Like "the Dustin" sounds wrong.
For example, "The primary `workflow.yaml` is designed ..." doesn't sound right. Rewording it to something like "The primary `workflow.yaml` workflow is designed ..." would be an improvement. Applies to shown lines 5 and 27/28. It appears to be done this way below this comment.

Reply:
I agree, @jaredoconnell.

I submit that the _name_ of the "primary workflow file" is `workflow.yaml`.  Therefore, line 5 should be phrased
```
The primary workflow, [`workflow.yaml`](workflow.yaml), is designed [...]
```
unless we want to omit the file name, in which case we get
```
The [primary workflow](workflow.yaml) is designed [...]
```

Similarly, lines 27-28, should read something like
```
This workflow is looped over by the outer workflow (`workflow.yaml`), [...]
```
although we could omit the file name, since it is described in the immediately-preceding bullet.